applications/serial_lte_modem/doc/CARRIER_AT_commands.rst

```diff
@@ -31,27 +31,27 @@ These may be the following:
 
 * ``#XCARRIEREVT: 1,<status>``
 
-  Request to set the modem to full functional mode.
+  Request to set the modem to full functional mode, i.e. ``AT+CFUN=1``.
```

Comment:
We don't use "i.e." and "e.g." in the documentation, so this will need to be opened up.
```suggestion
  Request to set the modem to full functional mode, equivalent to the ``AT+CFUN=1` AT command`.
```

Should be applied to all the lines where these are used.

Reply:
Not equivalent but must issue the AT command as mentioned.
Change to another wording?

Reply:
Ah, so it doesn't automatically do it. Then, would this be correct:
```suggestion
  Request to set the modem to full functional mode, such as using the ``AT+CFUN=1`` AT command.
```
Alternatively, if the AT command is basically the only option, we can change the order:
```suggestion
  Request to use the ``AT+CFUN=1`` AT command to set the modem to full functional mode.
```